tests/nonroot_test.go

```diff
@@ -36,7 +38,7 @@ var _ = Describe("[sig-compute]NonRoot feature", func() {
 				Skip(fmt.Sprintf("Missing %s, enable %s featureGate.", virtconfig.VirtIOFSGate, virtconfig.VirtIOFSGate))
 			}
 
-			vmi := tests.NewRandomVMIWithPVCFS("test")
+			vmi := libvmi.NewFedora(libvmi.WithFilesystemPVC("virtiofs", "test"))
```

Comment:
Why fedora?
It probably does not make a difference here.

Reply:
It was fedora before but actually it should make no difference. Let's try with `libvmi.New` instead.

Reply:
If it was fedora, maybe it needed guest agent? I would not change it in a PR that is supposed to be a refactoring. unless you are sure it will be much better.

Reply:
The test wants the creation of the VM to fail with a specific error message so presence of the guest agent should not matter.